### DISCUSSION TITLE
How to use React Skeleton with nextjs 14.4

### DISCUSSION THREAD

**Discussion by aldairandre** [dvtng/react-loading-skeleton#215] [Q&A]
Hello everyone, this is my first time using React Skeleton, and I ran into a difficulty using it, I'm trying to use React Skeleton with React's Suspense component but for some reason my Skeleton is not generated, could anyone tell me What would be the most practical way for me to implement React Skeleton using Nextjs?
im using Nextjs 14.3
and React Skeleton v3
Below is the code for my app

[My component image](https://github.com/dvtng/react-loading-skeleton/assets/72257475/f75fb9ff-5d8c-49a2-85c5-1554016d4253)

[Loading component image](https://github.com/dvtng/react-loading-skeleton/assets/72257475/21061e26-9809-46d3-a2a5-810b86559cea)

**Comment by srmagura** [comment]
The code you posted looks fine, so I am not sure what is wrong. If you post a runnable repro case, I can look at it (or you might find the issue while removing non-essential pieces of code).
